library/ecp.c

```diff
@@ -248,9 +248,16 @@ int mbedtls_ecp_check_budget( const mbedtls_ecp_group *grp,
         else if( grp->pbits >= 384 )
             ops *= 2;
 
-        /* avoid infinite loops: always allow first step */
-        if( rs_ctx->ops_done != 0 && rs_ctx->ops_done + ops > ecp_max_ops )
+        /* Avoid infinite loops: always allow first step.
+         * Because of that, however, it's not true that
+         * ops_done <= ecp_max_ops at any time, so the
```

Comment:
"at all times" or "at any time"? I would think at some times ops_done <= ecp_max_ops, but not all times.

Reply:
@Patater The implicit bracketing is `it's not true that (ops_done <= ecp_max_ops at any time)`, and not `(it's not true that ops_done <= ecp_max_ops) at any time`. I'm not sure how `at all times` would make things clearer. I'll switch to `Because of that, however, it's not generally true that ops_done <= ecp_max_ops, so the ...`.